lib/index.js

```diff
@@ -98,6 +98,16 @@ class WebpackPluginServe extends EventEmitter {
       options.static = [];
     }
 
+    if (options.waitForBuild) {
```

Comment:
why do we need a `done` handler here as well as on line 170? is it to catch the first build and then any subsequent builds?

Reply:
Yes